powersimdata/data_access/launcher.py

```diff
@@ -151,6 +151,8 @@ def check_progress(self):
 
 
 class HttpLauncher(Launcher):
+    BASE_URL = f"http://{server_setup.SERVER_ADDRESS}:5000"
+
```

Comment:
should we define `scenario_id` here since it is used in `_launch`, `extract_simulation_output` and `check_progress`?

Reply:
We could also have a constructor for the class where we set `scenario_id` and `base_url` as attributes of the class

Reply:
Done for `scenario_id`. Left the base url since it's static, so only needs to be defined for the class (not per instance), even though we access it using `self`.